src/elm/Community.elm

```diff
@@ -110,6 +110,8 @@ type alias Community =
     , memberCount : Int
     , members : List Profile
     , objectives : List Objective
+    , actions : Bool
+    , shop : Bool
```

Comment:
The same suggestion as in the [related backend branch](https://github.com/cambiatus/backend/pull/39#pullrequestreview-413844237): `hasActions` and `hasShop` will work better here.